notebooks/individual-dashboards/erroranalysis-dashboard/erroranalysis-dashboard-regression-housing.ipynb

```diff
@@ -7,7 +7,7 @@
     "\n",
     "# Analyze Errors and Explore Interpretability of Models\n",
     "\n",
-    "This notebook demonstrates how to use the Responsible AI Widget's Error Analysis dashboard to understand a model trained on the regression boston housing dataset. The goal of this sample notebook is to predict housing prices with scikit-learn and explore model errors and explanations:\n",
+    "This notebook demonstrates how to use the Responsible AI Widget's Error Analysis dashboard to understand a model trained on the regression california housing dataset. The goal of this sample notebook is to predict housing prices with scikit-learn and explore model errors and explanations:\n",
     "\n",
     "1. Train an SVM classification model using Scikit-learn\n",
```

Comment:
Do we also need to update the UI tests with the new notebook name? @vinuthakaranth could you help confirm?

Reply:
If we are making any changes to notebooks under folder -  https://github.com/microsoft/responsible-ai-toolbox/tree/main/notebooks/responsibleaidashboard, E2E tests has to be revisited. If not, no changes required from E2E perspective.